test_scripts/API/SubtleAlertStyle/commonSubtleAlert.lua

```diff
@@ -0,0 +1,140 @@
+local common = require('test_scripts/Smoke/commonSmoke')
+
+local commonSubtleAlert = common
+
+local function sendOnSystemContext(pCtx)
```

Comment:
Could make this a common function so that https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2433/files#diff-71129c961167b35d450a0771f9e67469R125 can be removed